test/HiGlassComponentTest.js

```diff
@@ -1802,57 +1801,58 @@ describe('Simple HiGlassComponent', () => {
 
 
     it('Moves the brush on one of the views', (done) => {
-      const heatmapTrack = getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
+      const track = getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
 
-      const domain1 = heatmapTrack.limitedValueScale.domain();
+      const domain1 = track.limitedValueScale.domain();
 
 
-      heatmapTrack.gColorscaleBrush.call(heatmapTrack.scaleBrush.move,
+      track.gColorscaleBrush.call(track.scaleBrush.move,
         [0, 100]);
 
-      const domain2 = heatmapTrack.limitedValueScale.domain();
+      const domain2 = track.limitedValueScale.domain();
 
       // we don't expect the other view to change
       expect(domain1[0]).not.toEqual(domain2[0]);
 
-
       // console.log('domain1:', domain1);
       // console.log('domain2:', domain2);
 
-
       done();
     });
 
     it('locks the scales and recenters the page', (done) => {
       hgc.instance().handleValueScaleLocked('aa', 'heatmap1', 'view2', 'heatmap2');
-      const track1 = getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
-      const track2 = getTrackObjectFromHGC(hgc.instance(), 'view2', 'heatmap2');
+      getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
+      getTrackObjectFromHGC(hgc.instance(), 'view2', 'heatmap2');
 
       // zoom out a little bit
-      hgc.instance().tiledPlots.aa.trackRenderer.setCenter(1799432348.8692136, 1802017603.5768778, 28874.21283197403);
+      hgc.instance().tiledPlots.aa.trackRenderer.setCenter(
+        1799432348.8692136, 1802017603.5768778, 28874.21283197403
+      );
 
       waitForTilesLoaded(hgc.instance(), done);
     });
 
     it('Moves the brush on one view and makes sure it moves on the other', (done) => {
-      const heatmapTrack = getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
+      const track = getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
 
       // console.log('lvs1', heatmapTrack.limitedValueScale.domain());
 
       // move the brush down to limit the amount of visible data
-      heatmapTrack.gColorscaleBrush.call(heatmapTrack.scaleBrush.move,
+      track.gColorscaleBrush.call(track.scaleBrush.move,
         [0, 100]);
 
       // console.log('lvs2', heatmapTrack.limitedValueScale.domain());
 
       const heatmap2Track = getTrackObjectFromHGC(hgc.instance(), 'view2', 'heatmap2');
 
-      expect(heatmapTrack.options.scaleStartPercent).toEqual(heatmap2Track.options.scaleStartPercent);
-      expect(heatmapTrack.options.scaleEndPercent).toEqual(heatmap2Track.options.scaleEndPercent);
+      expect(heatmapTrack.options.scaleStartPercent)
```

Comment:
After renaming the variable above from `heatmapTrack` to `track`, what does `heatmapTrack` here refer to?

Reply:
the import from view-configs